sql/_35_fig_cake/grant_revoke_redefine/cases/cbrd_25627_05.sql

```diff
@@ -0,0 +1,80 @@
+/* Verfified the CBRD-25627
+As-is: if changed the owner of objects then it is not updated on db_auth
+To-be: If changed the owner of objects then revoke all grantors
+*/
+
+
+
+create user u1;
+create user u2;
+create user u3;
+
+evaluate 'case 5: changed owner for partition table';
+
+evaluate 'connect to u1 & create u1.t1_r, u1.t1_h, u1.t1_l partition table';
+call login ('u1') on class db_user;
+create table t1_r (c1 int) comment 'u1 > t1_r (range)' 
+  partition by range (c1) (
+      partition p0 values less than (0) comment 'u1 > t1_r > p0 (range)',
+      partition p1 values less than maxvalue comment 'u1 > t1_r > p1 (range)'
+    );
+
+create table t1_h (c1 int) comment 'u1 > t1_h (hash)'
+  partition by hash (c1) partitions 2;
+
+create table t1_l (c1 int) comment 'u1 > t1_l (list)' 
+  partition by list (c1) (
+      partition p0 values in (0) comment 'u1 > t1_l > p0 (list)',
+      partition p1 values in (1) comment 'u1 > t1_l > p1 (list)'
+    );
+
+evaluate 'u1.t1_r, u1.t1_h, u1.t1_l grant to u2';
+grant select on t1_r to u2 with grant option;
+grant select on t1_h to u2 with grant option;
+grant select on t1_l to u2 with grant option;
+
+select grantor_name, grantee_name, owner_name, object_type, object_name, auth_type from db_auth where grantee_name != 'PUBLIC' order by grantor_name, object_name;
```

Comment:
```suggestion
select grantor_name, grantee_name, owner_name, object_type, object_name, auth_type, is_grantable from db_auth where grantee_name != 'PUBLIC' order by grantor_name, object_name;
```
with grant option 이 사용되었으므로 is_grantable 추가하는 것이 좋겠습니다.

Reply:
반영했습니다.
리뷰 감사합니다.